types/ejs/index.d.ts

```diff
@@ -279,6 +285,17 @@ export type RethrowCallback = (
  */
 export type IncludeCallback = (path: string, data?: Data) => string;
 
+/**
+ * @param originalPath the path as it appears in the include statement
+ * @param parsedPath the previously resolved path
+ *
+ * @return an object where {@link filename} is the final parsed path or {@link template} is the content of the included template
+ */
+export type IncluderCallback = (
+    originalPath: string,
+    parsedPath: string,
+) => { filename: string; template?: never } | { template: string; filename?: never };
```

Comment:
Can you make this into a type as well? (and call `IncluderResult` as in their original naming:
https://github.com/mde/ejs/blob/f21a9e464337032af5e61352c54b9dd8dae8fd1b/lib/ejs.js#L312

Reply:
Updated to reflect this suggestion